packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -211,7 +211,12 @@ export class DefaultCrudRepository<T extends Entity, ID>
       this.modelClass.findById(id, filter, options),
     );
     if (!model) {
-      throw new Error(`no ${this.modelClass.name} found with id "${id}"`);
+      const quotedId = JSON.stringify(id);
+      const err: Error & {code?: string} = new Error(
```

Comment:
Maybe declare the type `Error & {code?: string}` somewhere? We will be using it in other places too.

Reply:
Good point 👍